README.md

```diff
@@ -710,9 +709,8 @@ Other Style Guides
     }
 
     // good
-    let test;
     if (currentUser) {
-      test = () => {
+      const test = () => {
```

Comment:
Actually here I wanted to use `const` inside of the `if` block.

I thought that this point [7.4](https://github.com/airbnb/javascript/blob/master/README.md#functions--note-on-blocks) should answer the question -
_What if I need to create a function in the block?_

Answer: In such case don't use function declaration, create a variable using const (because in this good example we don't assign new value and therefore should use const) in the block and assign a function to it instead.

If I misunderstood idea, let's keep it as it is now.

Reply:
Sure, either of these are fine. how about adding your example, so both are present?